### ISSUE TITLE
Implement Academy CLF

### ISSUE THREAD

**Issue by scoopxyz** [imageworks/OpenColorIO#491]
Self-explanatory and much needed, add momentum here.

**Comment by hodoulp** [comment]
Part of OCIO v2

**Comment by dbr** [comment]
(CLF being [Common LUT Format](https://github.com/ampas/CLF))

**Comment by hodoulp** [comment]
@BernardLefebvre is working on that.

**Comment by michdolan** [comment]
#656 Added CLF read support. Sufficient to close this?

**Comment by BernardLefebvre** [comment]
I am still working on CLF write support.

**Comment by doug-walker** [comment]
You could close this as a duplicate of #610.  And as Bernard wrote, we were waiting on the writer to close that one.

**Comment by michdolan** [comment]
As Doug stated, closing this as duplicate of #610 .
